src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -234,13 +234,16 @@ public int InstallApk(string apkPath)
                 result = RunAdbCommand($"install \"{apkPath}\"");

```

Comment:
I think we should also detect the `Exception occurred while executing 'install'` string here, besides the `AdbDeviceFullInstallFailureMessage`

Reply:
Great point, I will add